src/goto-instrument/code_contracts.h

```diff
@@ -26,6 +28,8 @@ Date: February 2016
 #include <util/namespace.h>
 #include <util/pointer_expr.h>
 
+#include "loop_utils.h"
```

Comment:
Wouldn't forward declarations do the trick?

Reply:
Sorry, resolved this comment by mistake without addressing it.

I `#include`d this header file for the `loopt` type, which is `typedef`ed as:

```c
typedef const natural_loops_mutablet::natural_loopt loopt;
```

Could I forward declare it here somehow? I wasn't sure how to.

Reply:
I believe one of `typename loopt;` or `class loopt;` might still work, and likely the compiler will telly you if it doesn't :-) Not high priority, though.